docs/tutorials/mlflow.ipynb

```diff
@@ -0,0 +1,774 @@
+{
```

Comment:
<p>Why aren't we adding this class to MLRun? There doesn't seem to be anything specific about it, won't you always want to use this class when using a model created via MLFlow run?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/4694/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>yes you would, it's the generic ModelServer, will soon be in mlrun hopefully </p>